### PR TITLE
Qml vsbuild

### PR DESCRIPTION
**Pull request by btfields**
Flatten sleek theme file structure and update imports so that the contents of the 'sleek' directory can be added directly into Qt Resources in the VS build and replace the default theme compiled into the plugin.  The changes don't affect the sleek theme files being used at runtime as they were before we replaced the default theme.

### CODE REVIEW

**Comment by sclem** [comment]
Looks good!

File: player/themes/sleek/Settings.qml [hunk 1]
```diff
@@ -9,7 +9,7 @@ QtObject {
 	property var dragging: false;
 	property var ismoving: 1;
 	property var buffering: 0;
-	property var toolbar: 1;
+	property var toolbar: 0;
 	property var uiVisible: 1;
```
Comment (sclem) [inline player/themes/sleek/Settings.qml]: Maybe setting this to 0 will hide the toolbar in fullscreen?

Comment (cdated) [inline player/themes/sleek/Settings.qml]: This has the toolbar:0, if someone else can try it out: https://drive.google.com/open?id=0B4nWEFBB2RgCUXZHT01JcVVabTQ

**Comment by SeanThomasWilliams** [comment]
Good

**Pull request merged**